core/src/main/scala/org/apache/spark/sql/rapids/tool/util/RuntimeUtil.scala

```diff
@@ -73,6 +82,25 @@ object RuntimeUtil extends Logging {
       "os.version" -> System.getProperty("os.version")
     )
   }
-}
-
 
+  def getJVMHeapArguments: Map[String, String] = {
+    ManagementFactory.getRuntimeMXBean.getInputArguments.filter(
+      p => p.startsWith("-Xmx") || p.startsWith("-Xms") || p.startsWith("-XX:")).map {
+      sizeArg =>
+        if (sizeArg.startsWith("-Xmx")) {
+          ("jvm.arg.heap.max", sizeArg.drop(4))
+        } else if (sizeArg.startsWith("-Xms")) {
+          ("jvm.arg.heap.min", sizeArg.drop(4))
+        } else { // this is heap argument
```

Comment:
nit: Should the comment say `this is gc argument` as the block contains arguments for GC?